### ISSUE TITLE
"Checking Package Dependencies" emits an obscure warning with BiocCheck 1.31.36

### ISSUE THREAD

**Issue by hpages** [Bioconductor/BiocCheck#154]
```
> BiocCheck("RAREsim_0.99.1.tar.gz")
This is BiocCheck version 1.31.36. BiocCheck is a work in progress.
Output and severity of issues may change. Installing package...
* Checking Package Dependencies...
Warning in system2(cmd, args, stdout = TRUE, stderr = FALSE, env = "R_DEFAULT_PACKAGES=NULL") :
  running command 'R_DEFAULT_PACKAGES=NULL '/home/hpages/R/R-4.2.r81969/bin/R' -q --vanilla --slave -f /home/hpages/R/R-4.2.r81969/library/BiocCheck/script/checkBadDeps.R --args "/home/hpages/pkgreviews/RAREsim.expressreview/RAREsim_0.99.1.tar.gz" "/tmp/RtmpbYw1B6/file4171c7609d77e/lib" 2>/dev/null' had status 1
```
I'm not sure how to interpret this warning. Will probably confuse contributors and most reviewers.

Also note that the use of caps is inconsistent with other checks:
```
* Checking Package Dependencies...
...
...
* Checking for deprecated package usage...
...
```

Thanks,
H.

**Comment by LiNk-NY** [comment]
This is an artifact of R-devel and it is fixed in the `output` branch which will be merged soon. 
https://github.com/Bioconductor/Contributions/issues/2585#issuecomment-1097110577

Best,
Marcel

**Comment by LiNk-NY** [comment]
I'll leave it open until it is merged.

**Comment by LiNk-NY** [comment]
closed in 16cf48c667858f9ce7dadefe8f69140c9efcfd91
